backend/src/admin/admin.controller.ts

```diff
@@ -21,17 +23,11 @@ export class AdminController {
     return this.adminService.login(userData);
   }
 
-  /** TEST CODE STARTS */
-  @Get('t1')
-  async getAll() {
-    return this.adminService.getAll1();
-  }
-
+  @UsePipes(new ValidationPipe())
   @UseGuards(AuthGuard('jwt'))
-  @Get('t2')
-  async getAll2() {
-    return this.adminService.getAll2();
+  @Post('register/junior')
+  async registerJunior(@Admin() payload: any, @Body() userData: RegisterJuniorDto) {
+    await this.adminService.verifyIsAdmin(payload.user);
```

Comment:
This is related to the comment in the PR? I would add a TODO so it's not forgotten.

Reply:
Adding Todos